finetuning/generalists/training/electron_microscopy/boundaries/obtain_boundaries_em_datasets.py

```diff
@@ -0,0 +1,130 @@
+import os
+import numpy as np
+
+from elf.io import open_file
+
+from skimage.measure import label
+from skimage.segmentation import watershed
+
+from torch_em import get_data_loader
+from torch_em.transform.label import label_consecutive
+from torch_em.transform.label import PerObjectDistanceTransform
+from torch_em.data import ConcatDataset, MinInstanceSampler, datasets
+
+from micro_sam.training import identity
+from micro_sam.training.util import ResizeRawTrafo, ResizeLabelTrafo
+
+
+def compute_platy_rois(root, sample_ids, ignore_label, file_template, label_key):
+    rois = {}
+    for sample_id in sample_ids:
+        path = os.path.join(root, (file_template % sample_id))
+        with open_file(path, "r") as f:
+            labels = f[label_key][:]
+        valid_coordinates = np.where(labels != ignore_label)
+        roi = tuple(slice(
+            int(coord.min()), int(coord.max()) + 1
+        ) for coord in valid_coordinates)
+        rois[sample_id] = roi
+    return rois
+
+
+def axondeepseg_label_trafo(labels):
+    # after checking, labels look like this : 0 is bg, 1 is myelins and 2 is axons
+    foreground_seeds = label((labels == 2))
+    boundary_prediction = (labels == 1)
+    seg = watershed(boundary_prediction, markers=foreground_seeds, mask=(foreground_seeds + boundary_prediction) > 0)
+    seg = label_consecutive(seg)
```

Comment:
Applying label consecutive here should not be necessary. The seeds and corresponding watershed will already be consecutive.

Reply:
True, makes sense. Thanks!